CONTRIBUTING.adoc

```diff
@@ -126,46 +127,71 @@ To facilitate rapid development we've put together a Vagrantfile you can use to
 
         $ sudo chmod +r openshift.local.config/master/openshift-registry.kubeconfig
         $ sudo chmod +r openshift.local.config/master/admin.kubeconfig
-        $ ./oadm registry --create --credentials=openshift.local.config/master/openshift-registry.kubeconfig --config=openshift.local.config/master/admin.kubeconfig
+        $ oadm registry --create --credentials=openshift.local.config/master/openshift-registry.kubeconfig --config=openshift.local.config/master/admin.kubeconfig
 
-10.  You are now ready to edit the source, rebuild and restart OpenShift to test your changes.
 
-11.  NOTE:  to properly stop OpenShift and clean up, so that you can start fresh instance of OpenShift, execute:
+10.  At this point it may be helpful to create a registry and load some image streams and templates.  The following commands will do so. These commands will make use of fixtures from the `openshift/origin/examples` dir:
 
-        $ sudo pkill openshift
-        $ docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
-        $ mount | grep "openshift.local.volumes" | awk '{ print $3}' | xargs -l -r sudo umount
-        $ cd <to the dir you ran openshift start> ; sudo rm -rf openshift.local.*
-        $ mount | grep < "\/absolute\/path", where that is the value passed to the `--volume-dir` argument on openshift start >  | awk '{ print $3}' | xargs -l -r sudo umount
-        $ sudo rm -rf < "\/absolute\/path", where again that is the  value passed to the `--volume-dir`argument >
+        # load image stream
+        $ oc create -f /data/src/github.com/openshift/origin/examples/image-streams/image-streams-centos7.json -n openshift --config=openshift.local.config/master/admin.kubeconfig
+        # load templates
+        $ oc create -f /data/src/github.com/openshift/origin/examples/sample-app/application-template-stibuild.json -n openshift --config=openshift.local.config/master/admin.kubeconfig
+        $ oc create -f /data/src/github.com/openshift/origin/examples/db-templates --config=openshift.local.config/master/admin.kubeconfig
 
-TIP: To ensure you get the latest image.  First run `vagrant box remove fedora_inst`.  And if later on you employ a dev cluster, additionally run  `vagrant box remove fedora_deps`.
+
+11. At this point you can open a browser on your host system and navigate to https://localhost:8443/console to view the web console.
+
+
+12.  NOTE:  to properly stop OpenShift and clean up, so that you can start fresh instance of OpenShift, execute:
+
+	# shut down openshift
+	$ sudo pkill openshift
+	# stop the docker containers
+	$ docker ps | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r docker stop
+	# deleting all the internal config files, etcd, etc and starting openshift fresh
+	sudo rm -rf openshift.local.*
+	# if you used the --volume-dir=/home/vagrant/volumes flag, then run these
+	$ mount | grep /home/vagrant/volumes | awk '{ print $3}' | xargs -l -r sudo umount
+	$ sudo rm -rf /home/vagrant/volumes
```

Comment:
Can probably take these lines out since we removed volume-dir from the instructions.